### DISCUSSION TITLE
firmware for the ENDER 3 S1 PRO

### DISCUSSION THREAD

**Discussion by asabache** [mriscoc/Ender3V2S1#1061] [General]
hello, greetings from venezuela, is there a possibility that you make a firmware for the ENDER 3 S1 PRO? This same one that I use in my ender 3 V2 and it is excellent, I updated my S1 PRO to the most recent version that Crelity has, SWV2.0.8.28F4_F401, but it is not even half as good as yours.

**Comment by mriscoc** [comment]
Look in the wiki for firmware requirements
